components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -155,6 +155,8 @@ spec:
             value: "file"
         {{- end}}
         {{- if .Values.global.remoteControlPlane.enabled }}
+          - name: TYK_GW_ENABLEANALYTICS
+            value: "{{.Values.gateway.analyticsEnabled}}"
```

Comment:
should this be applicable for all but not only when remoteControlPlane is enabled?